lib/evmone/baseline.cpp

```diff
@@ -8,6 +8,7 @@
 #include "execution_state.hpp"
 #include "instructions.hpp"
 #include "vm.hpp"
+#include <iostream>
```

Comment:
Is this still needed?

Reply:
`std::cerr << "EOF validation failed: " << get_error_message(err) << '\n';`
yes. this line uses it.

Reply:
There shouldn't be `cerr` in `baseline`, imho.